### ISSUE TITLE
Occasional error 125 on random tests

### ISSUE THREAD

**Issue by icecrime** [docker/docker#21473]
Seen twice on `win2lin` so far:

- [24020](https://jenkins.dockerproject.org/job/Docker-PRs-Win2Lin/24020/console)

```
03:45:04 FAIL: docker_cli_events_test.go:93: DockerSuite.TestEventsLimit
03:45:04 
03:45:04 docker_cli_events_test.go:115:
03:45:04     c.Assert(err, checker.IsNil, check.Commentf("%q failed with error", strings.Join(args, " ")))
03:45:04 ... value *exec.ExitError = &exec.ExitError{ProcessState:(*os.ProcessState)(0xc082300220)} ("exit status 125")
03:45:04 ... "run --rm busybox true" failed with error
```

- [24017](https://jenkins.dockerproject.org/job/Docker-PRs-Win2Lin/24017/console)

```
00:16:24 FAIL: docker_cli_run_test.go:755: DockerSuite.TestRunTwoConcurrentContainers
00:16:24 
00:16:24 docker_cli_run_test.go:785:
00:16:24     c.Assert(err, check.IsNil)
00:16:24 ... value *exec.ExitError = &exec.ExitError{ProcessState:(*os.ProcessState)(0xc082568060)} ("exit status 125")
```

**Comment by mlaventure** [comment]
For `DockerSuite.TestEventsLimit`, the daemon error message is as follow:

> Container 4 failed: exit status 125 -- C:\CI\CI-9b7ac1e\docker.exe: Error response from daemon: error creating aufs mount to /var/lib/docker/aufs/mnt/6ceec7f8446e85076d04f8d54be673bb131d8c09fad1626d020ed0f9451da10f: invalid argument. 
See 'C:\CI\CI-9b7ac1e\docker.exe run --help'.

**Comment by tonistiigi** [comment]
This was likely fixed by #21552
